.github/CODEOWNERS

```
@@ -8,12 +8,13 @@
 # for every PR that updates stacks.
 # 2. As you may notice some stacks are owned by the devfile/devfile-services-team
 # directly.
-# 3. All the owner mentions follow the format below:  
+# 3. All the owner mentions follow the format below:
 # <path-to-the-stack-dir> @owner @reviwer-team
 
 # .Net stacks
 /stacks/dotnet50/       @kadel @devfile/che-team
 /stacks/dotnet60/       @kadel @devfile/che-team
+/stacks/dotnet80/       @msivasub @devfile/che-team
```

@msivasubramaniaan I think your handle is different one right?

---

Yes I am from IDE Raptors. Shall I update with `/stacks/dotnet80/  @msivasub @IDERaptors`?

---

No, as mentioned at the top of the file, the `che-team` is a required reviewer for all stack related PRs so we should keep them there. My comment is for your own handle I think the current one is not the same as yours right? So suggested change is:

```suggestion
/stacks/dotnet80/       @msivasubramaniaan @devfile/che-team
```